CONTRIBUTING.md

```diff
@@ -1,15 +1,158 @@
 # Contributing to Zanata
 
+We would love for you to contribute to our source code and to make Zanata
+even better than it is today! Here are the guidelines we would like you to
+follow:
```

Comment:
We could emphasize that following these guidelines will make sure that we see their feedback and they will have the best experience when contributing code. e.g.

```
...
even better than it is today! To make sure we see your feedback and contributions straight away,
please follow these guidelines:
```